<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page3</class>
 <widget class="QWidget" name="Page3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>APPID  </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inputAppid">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
  border: 1px solid #d3d3d3;
  border-radius: 5px;
  padding: 5px;
  background-color: #ffffff;
  font-size: 14px;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>APPKEY</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inputAppkey">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
  border: 1px solid #d3d3d3;
  border-radius: 5px;
  padding: 5px;
  background-color: #ffffff;
  font-size: 14px;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="determineButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #f6f7fa, stop:1 #dadbde);
  border: 1px solid #bdc3c7;
  border-radius: 5px;
  padding: 5px;
  font-size: 14px;
  text-align: center;
}

QPushButton:hover {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #e0e0e0, stop:1 #cecece);
}

QPushButton:pressed {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #dadbde, stop:1 #f6f7fa);
  padding-left: 5px;
  padding-top: 5px;
}

QPushButton:disabled {
  background-color: #f6f7fa;
  border: 1px solid #bdc3c7;
  color: #bdc3c7;
}
</string>
        </property>
        <property name="text">
         <string>上传</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>获取方法请查看目录下帮助文档</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
